app/assets/javascripts/settings/index.js

```diff
@@ -11,7 +11,7 @@ $(function() {
     new RegistryForm($registryForm);
   }
 
-  if ($systemCertificateForm.length) {
-    new SystemCertificateForm($systemCertificateForm);
```

Comment:
There's no `SystemCertificateForm`. I let this pass on a previous PR. Removed it since it was causing a js error.